tests/features/search/search.spec.ts

```diff
@@ -0,0 +1,139 @@
+import { expect } from '@playwright/test';
+import { test } from '../../config/test-base';
+
+const bookETLDataEntity = 'Book_ETL_aqa';
+const specialCharactersDataEntity = '_AirFlights';
+const numbersDataEntity = '737boeing';
+
+test.describe('Search', () => {
+  test.beforeEach(async ({ steps: { pages }, page }) => {
+    await test.step(`I open Catalog page`, async () => {
+      await page.goto('');
+      await pages.topPanel.clickTab('Catalog');
+    });
+  });
+  test.describe('Search by single word', () => {
```

Comment:
What about naming cases in the following manner?

- When searching for a single keyword
    - Should display expected item when entering a valid expression
    - Should display an empty list when entering an invalid expression
    - Should display expected item when entering name which starts with special chars
    - Should display expected item when entering name which starts with numbers
    - Should display full list when entering an empty string

- When searching for multiple keywords
    - Should display expected item when entering both expressions are contained in a single entity
    - Should display an empty list when entering expressions that are part of different entities
    - Should display expected item when entering name which starts with numbers
    - Should display an empty list when entering 1 valid and 1 invalid expression
    - Should display an empty list when entering 2 invalid expressions